### PR TITLE
December '21 events

### PR DESCRIPTION
**Pull request by bobturneruk**
Text is mostly verbatim from organisers.

### CODE REVIEW

File: _events/seminar-2021-12-02.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+image:
+slides_url:
+---
+
+**This event is being run by the Testing Research Group**
+
+Software developers use Stack Overflow to interact and exchange code snippets and research uses Stack Overflow to harvest code snippets for use with recommendation systems. However, code on Stack Overflow may have quality issues, such as security or license problems. In this talk I will present our work on how users of Stack Overflow perceive such issues, how we studied the problem of outdated code and potential license violations, and how code is non-compliant with coding style. The prevalence of such toxic code may affect mining or machine learning tasks using Stack Overflow as a code base.
+
+Google meet link: <https://meet.google.com/zbx-hhot-ucp>
```
Comment (mondus) [inline _events/seminar-2021-12-02.md:21]: Not sure we want to put the google meet link on a public site. I think this has probably only been shared via internal email so far. Perhaps an email address to contact the seminar organiser instead?

**Review by mondus — CHANGES_REQUESTED** [review]

File: _events/seminar-2021-12-02.md (hunk 1 above)
Comment (bobturneruk) [reply _events/seminar-2021-12-02.md:21]: We do this habitually for LunchBytes - the organiser is OK with it. I have my reservations, especially if they've not configured it to block non-Sheffield by default.

Comment (bobturneruk) [reply _events/seminar-2021-12-02.md:21]: Actually he says he'll have to let them into the call - do you think this is OK @mondus? I'm alright with it.

Comment (mondus) [reply _events/seminar-2021-12-02.md:21]: As long as they have said they are ok with it then it is fine. Its then their own responsibility of they get googlebombed. Is that the google meet equivalent word?

**Review by mondus — APPROVED** [review]

**Pull request merged**